offchain/Dockerfile

```
@@ -34,7 +34,7 @@ COPY . /usr/src/app
 RUN cargo build --release
 
 # define runtime image
-FROM debian:bookworm-20230725-slim as runtime
+FROM debian:bookworm-20230814-slim as runtime
```

Could we use the latest version instead of updating it regularly?
```suggestion
FROM debian:bookworm-slim as runtime
```

---

We could, but it would make our builds less predictable.

---

Debian is reasonably stable; it will not impact the predictability much.

---

I will approve this now; we can discuss this again for the next release.